lib/exportPlatform.js

```diff
@@ -41,9 +42,17 @@ function exportPlatform(platform) {
   // values like "1px solid {color.border.base}" we want to
   // transform the original value (color.border.base) before
   // replacing that value in the string.
-  return resolveObject(
+  to_ret = resolveObject(
     transformObject(this.properties, platformConfig)
   );
+
+  var ERROR_GROUP = 'Property Reference Errors';
```

Comment:
Can we just have these error group types on the ErrorHandler object itself, like ErrorHandler.referenceErrors or something so we don't have to keep defining ERROR_GROUP in different files and making sure their strings match up?